src/main/java/gtPlusPlus/xmod/gregtech/common/blocks/GregtechMetaSpecialMultiCasings.java

```diff
@@ -71,7 +71,6 @@ public GregtechMetaSpecialMultiCasings() {
         GregtechItemList.Casing_Turbine_HP.set(new ItemStack(this, 1, 2));
         GregtechItemList.Casing_Turbine_Gas.set(new ItemStack(this, 1, 3));
         GregtechItemList.Casing_Turbine_Plasma.set(new ItemStack(this, 1, 4));
-        GregtechItemList.Casing_TeslaTower.set(new ItemStack(this, 1, 5));
```

Comment:
I only removed this reference, the casing is still visible in NEI but I didn't want to mess too much with this class.